### DISCUSSION TITLE
Breakglass Password Stored in Secrets Manager

### DISCUSSION THREAD

**Discussion by jeixav** [aws-samples/aws-secure-environment-accelerator#1083] [Q&A] [answered]
I notice the passwords for bgUser1 and bgUser2 are stored in AWS Secrets Manager. Is this required? For what purpose?

I'm trying to understand if deleting those secrets might break some ASEA feature.

**Comment by Brian969** [comment, marked as answer]
Yes, first time use passwords for IAM users are stored in secrets manager of the management account.  It is expected that customers will retrieve the value of the passwords from secrets manager for their first use and then change the passwords as appropriate.  The new value does NOT need to be stored back into the secret (except for the adconnector-usr, which must be updated after changing).  I would not recommend deleting the secrets (which contain meaningless information after 1st use) as this is likely to break the CloudFormation.
